common/test/acceptance/pages/studio/settings_advanced.py

```diff
@@ -227,5 +227,6 @@ def expected_settings_names(self):
             'instructor_info',
             'create_zendesk_tickets',
             'ccx_connector',
-            'enable_ccx'
+            'enable_ccx',
+            'enable_subsection_grades_saved',
```

Comment:
How do we feel about this practice in general? I always try to keep the trailing comma in lists, as it makes future diffs cleaner (one line vs two).